host/xtest/pkcs11_1000.c

```diff
@@ -64,6 +65,39 @@ static CK_MECHANISM cktest_hmac_sha384_mechanism = {
 static CK_MECHANISM cktest_hmac_sha512_mechanism = {
 	CKM_SHA512_HMAC, NULL, 0,
 };
+
+static CK_ULONG cktest_general_mechanism_hmac_len = 8;
+static CK_MECHANISM cktest_hmac_general_md5_mechanism = {
```

Comment:
prefer with an empty line between `CK_ULONG` length and the `CK_MECHANISM` structs.

could you add a `const` attribute. This length data is referenced at several places. Make sure has the expected value.

Reply:
ok